python-frontend/src/main/java/org/sonar/python/semantic/Scope.java

```diff
@@ -151,7 +151,7 @@ private Symbol copySymbol(String symbolName, Symbol symbol) {
         .collect(Collectors.toSet());
       return new AmbiguousSymbolImpl(symbolName, symbol.fullyQualifiedName(), alternativeSymbols);
     } else if (symbol.is(Symbol.Kind.OTHER)) {
-      SymbolImpl res = new SymbolImpl(symbolName, symbol.fullyQualifiedName());
+      SymbolImpl res = new SymbolImpl(symbolName, symbol.fullyQualifiedName(), ((SymbolImpl) symbol).annotatedType());
```

Comment:
I think we should also use this constructor in ProjectLevelSymbolTable.
https://github.com/SonarSource/sonar-python/blob/e6dd6dd1bbde8e461040bf59250670e6408a4ffa/python-frontend/src/main/java/org/sonar/python/semantic/ProjectLevelSymbolTable.java#L76